packages/tradershub/src/components/Modal/Modal.classnames.ts

```diff
@@ -2,7 +2,7 @@ import { cva, VariantProps } from 'class-variance-authority';
 import { ExcludeAllNull } from '@deriv/quill-design';
 
 export const ModalFooterClass = cva(
-    'grid grid-cols-2 gap-400 p-800 border border-solid border-t-100 border-system-light-secondary-background bottom-0 lg:flex lg:items-center lg:px-1200 lg:py-800',
+    'grid grid-cols-1 gap-400 p-800 border border-solid border-t-100 border-system-light-secondary-background bottom-0 lg:grid-cols-2 lg:flex lg:items-center lg:px-1200 lg:py-800',
```

Comment:
```suggestion
    'grid grid-cols-1 gap-400 p-800 border border-solid border-t-100 border-system-light-secondary-background bottom-0 lg:grid-cols-2 lg:items-center lg:px-1200 lg:py-800',
```

Reply:
Sorry just saw this. In the class there's flex and grid because we are using grid let's remove flex